packages/fast-usdc/src/exos/settler.js

```diff
@@ -256,7 +257,12 @@ export const prepareSettler = (
           const { value: fullValue } = fullAmount;
           const key = makeMintedEarlyKey(forwardingAddress, fullValue);
           if (mintedEarly.has(key)) {
-            mintedEarly.delete(key);
+            const count = mintedEarly.get(key);
```

Comment:
we should be using `get` and `delete` abstraction on the map to treat it as a multiset.

if we don't have a multiset durable store, we can have a utility that performs the multi-set on them.